src/components/Footer/__test__/Footer.test.js

```diff
@@ -0,0 +1,7 @@
+import renderer from "react-test-renderer";
+import Footer from "../Footer";
```

Comment:
Or to get rid of the warnings we don't use relative imports instead we use alias import like so : `import Footer from @components/Footer/Footer`